src/toil/resource.py

```diff
@@ -378,8 +393,10 @@ def forModule(cls, name: str) -> Any:
         method assumes that the module with the specified name has already been loaded.
         """
         module = sys.modules[name]
-        filePath = os.path.abspath(module.__file__)
-        filePath = filePath.split(os.path.sep)
+        fileAbsPath = ""
+        if module.__file__ is not None:
```

Comment:
We might want to throw a specific error if `__file__` is None. Right now it will complain that the (nonexistent) file doesn't end in `.py`.